File: examples/hello.go
// Ejemplo mínimo para outline / LSP gopls en tuide.
// Compilar: go build -gcflags="all=-N -l" -o hello hello.go

package main

import "fmt"

func greet(name string) {
	fmt.Printf("hello, %s\n", name)
}

func main() {
	greet("tuide")
	greet("sdf")
}

File: examples/hello.zig
// Ejemplo mínimo para outline / LSP zls en tuide.
// Compilar: zig build-exe -ODebug hello.zig

const std = @import("std");

fn greet(name: []const u8) void {
    std.debug.print("hello, {s}\n", .{name});
}

pub fn main() !void {
    greet("tuide");
}